packages/twenty-server/src/engine/core-modules/workspace/workspace.resolver.ts

```
@@ -190,6 +193,20 @@ export class WorkspaceResolver {
     }
   }
 
+  @ResolveField(() => RoleDTO, { nullable: true })
+  async defaultRole(@Parent() workspace: Workspace): Promise<RoleDTO | null> {
+    if (!workspace.defaultRoleId) {
```

should we throw if permissions are enabled and defaultRoleId is missing?